packages/api-page-builder-import-export/src/client.ts

```diff
@@ -13,35 +13,9 @@ export async function invokeHandlerClient<TParams>({
     payload,
     description
 }: InvokeHandlerClientParams<TParams>) {
-    /*
-     * Prepare "invocationArgs", we're hacking our wat here.
-     * They are necessary to setup the "context.pageBuilder" object among other things in IMPORT_PAGE_FUNCTION
-     */
-    const { request } = context;
-
-    const tenantId = context.tenancy.getCurrentTenant().id;
-
-    const headers = {
-        ...request.headers,
-        ["x-tenant"]: request.headers["x-tenant"] || tenantId
-    };
-    delete headers["content-length"];
-    const invocationArgs = {
-        httpMethod: request.method,
-        body: request.body,
-        headers,
-        /**
-         * Required until type augmentation works correctly.
-         */
-        cookies: (request as any).cookies
-    };
-    // Invoke handler
     await context.handlerClient.invoke<TParams & any>({
         name: name,
-        payload: {
-            ...payload,
-            ...invocationArgs
-        },
```

Comment:
@neatbyte-vnobis can you please explain how does code removal solve the issue?
I assume it has something to do with our migration to `fastify` some time ago, but maybe you can give us more details? Thanks!

Reply:
Ok, thanks. I mean, if this works, then great. The original code was an ugly hack to being with, so, I'm glad if we can remove that.

Reply:
I'll need to test this on a multi-tenant system, because this `x-tenant` header is important for the import/export Lambda to work on a correct tenant. If we just remove it from headers, it might be an issue.

Reply:
@neatbyte-vnobis I just tested this, and to make this work on a multi-tenant system, we need to have `headers` and `httpMethod` in the payload. Like this:

```
        payload: {
            ...payload,
            headers: {
                ["x-i18n-locale"]: request.headers["x-i18n-locale"],
                ["x-tenant"]: request.headers["x-tenant"] || tenantId
            },
            httpMethod: request.method
        }
```